features/api_token_info.feature

```diff
@@ -143,9 +104,38 @@ Feature: api.u.pro.token_info.v1
       """
 
     Examples: ubuntu release
-      | release | machine_type  |
-      | xenial  | lxd-container |
-      | bionic  | lxd-container |
-      | focal   | lxd-container |
-      | jammy   | lxd-container |
-      | noble   | lxd-container |
+      | release  | machine_type  |
+      | xenial   | lxd-container |
+      | bionic   | lxd-container |
+      | focal    | lxd-container |
+      | jammy    | lxd-container |
+      | noble    | lxd-container |
+      | oracular | lxd-container |
+
+  @uses.config.contract_token_staging_expired
+  Scenario Outline: Get token info from api
```

Comment:
Let's rename this scenario to better reflect what we are testing

Reply:
Thanks a lot for the constant help with the review @lucasmoura :sweat_smile: